language/transaction-builder/generator/README.md

```diff
@@ -60,7 +66,7 @@ target/debug/generate-transaction-builders \
     --with-diem-types "testsuite/generate-format/tests/staged/diem.yaml" \
     --target-source-dir "$DEST" \
     --with-custom-diem-code language/transaction-builder/generator/examples/python3/custom_diem_code/*.py -- \
-    "language/diem-framework/compiled/legacy/transaction_scripts/abi"
+    "language/diem-framework/compiled"
```

Comment:
could this be last time we change this?

Reply:
It should be! This is the final switchover that handles both the legacy transaction scripts and their generation, and also handles generation for the new script function.